config/webhook/manifests.yaml

```diff
@@ -125,6 +125,23 @@ webhooks:
     - CREATE
     resources:
     - persistentvolumeclaims
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validating-v1-registry
+  failurePolicy: Fail
```

Comment:
We cannot set the failurePolicy to Fail otherwise we could end up in a recursion loop.

Let's imagine the installation of Capsule along with this Validating Webhook: the Capsule Pod would need to pass through this but that wouldn't be yet ready since hasn't started yet.

It's something we already had with the Service resource annotations/labels webhook made by @maxfedotov

Reply:
makes sense, will change to ignore